python_modules/dagster/dagster/_core/asset_graph_view/entity_subset.py

```diff
@@ -104,6 +104,12 @@ def compute_parent_subset(self, parent_key: AssetKey) -> "EntitySubset[AssetKey]
     def compute_child_subset(self, child_key: U_EntityKey) -> "EntitySubset[U_EntityKey]":
         return self._asset_graph_view.compute_child_subset(child_key, self)
 
+    @cached_method
+    def compute_mapped_subset(
+        self: "EntitySubset[AssetKey]", to_key: U_EntityKey
```

Comment:
We can allow this method on any EntitySubsets, so it should be fine to drop the annotation for `self`